proposals/multiple_user_script_worlds.md

```diff
@@ -115,9 +120,10 @@ Relevant methods and types:
 
      /**
       * The list of ScriptSource objects defining sources of scripts to be
-      * injected into matching pages.
+      * injected into matching pages. This property must be specified for
+      * ${ref:register}
       */
-     js: ScriptSource[];
+     js?: ScriptSource[];
```

Comment:
There is consensus among the browser vendors that this should be optional. Devlin also confirmed that on behalf of Google.

I filed an issue for Chromium to change that, at https://issues.chromium.org/issues/383712209